contracts/truefi/LoanFactory.sol

```diff
@@ -46,6 +48,10 @@ contract LoanFactory is ILoanFactory, Initializable {
         lender = 0x16d02Dc67EB237C387023339356b25d1D54b0922;
     }
 
+    function setTru() external {
+        tru = 0x4C19596f5aAfF459fA38B0f7eD92F11AE6543784;
```

Comment:
If we are harcoding it anyway, and change will require update anyway, maybe let's not use setter, but just implement getter returning `tru` address